### DISCUSSION TITLE
Poll: Heat Set Inserts

### DISCUSSION THREAD

**Discussion by jon-harper** [jon-harper/OmniBox#36] [Polls]
I use [heat set inserts](https://www.amazon.com/iplusmile-Embedment-Threaded-Printing-Projects/dp/B087NBYF65) extensively in other projects. I've held off using them so far in this project.

![image](https://user-images.githubusercontent.com/11852387/193064260-c6c369c0-f5d7-4b87-bf54-44a34187c946.png)

Inserts became much more common in the past year or so, and I'd like to gauge interest in making them an option when printing your case.

## Effects

If inserts are taken up, they will likely be exclusively M3 inserts. Panels and trays that currently use M4 would switch to M3 screws, (EDIT) which is *not* a breaking change.

**Edit:** Any M3 screws, it turns out, have heads large enough to grab the old panels.

## Timeline

I do not have an end date in mind for this poll, aside from before 1.0 is released (which is not imminent).

**Comment by TheoBromine2022** [comment]
Not familiar enough with the inserts to make an opinion but I'd be willing to learn how to use them.

**Comment by florence-wolfe** [comment]
Beyond the M3/M4 screw change, do heat set inserts have an impact on the design and dimensions of the case? Other than the thickness of the walls to know how tall of an inset to use, I'm uncertain how much of a difference it makes to mention these other than in a footnote somewhere in the documentation.

**Reply by jon-harper** [reply]
> Beyond the M3/M4 screw change, do heat set inserts have an impact on the design and dimensions of the case?

No changes other than the dimensions of the holes. The switch to M3 is actually to keep the inserts from requiring more changes (the M3 insert holes are about the diameter of an M4 screw).

The only difference in the Bill of Materials are the inserts, as well.

Somewhere there's a line between too many inserts and not enough, as the inserts do add to overall cost and some parts are never disassembled to need them. This poll is my attempt to find that line. 🙂

**Reply by levelfungames** [reply]
I'm ok with or without inserts.  If you added them down the line, I would use them here or there, depending on the printing tolerance of parts without them, meaning I'd print the versions that had heated inserts for parts that were not super thick or parts that held heavier parts to them.

**Reply by levelfungames** [reply]
Something like the handle, especially, would benefit from heat inserts.  Just thought of it while attempting to attach it lol

**Reply by jon-harper** [reply]
That makes great sense! Thanks!

**Reply by levelfungames** [reply]
Mission accomplished. Heat set handle achieved.

**Comment by levelfungames** [comment]
![image](https://user-images.githubusercontent.com/115413736/200184369-39e351b4-3208-4044-bd8f-0e5c6ec08720.jpg)

**Reply by jon-harper** [reply]
That's both awesome and hilarious.

I am printing a "formal" heat set insert handle right now...but I think this one's a keeper.
